Rakefile

```diff
@@ -27,6 +27,16 @@ RSpec::Core::RakeTask.new(:acceptance) do |t|
     t.pattern = 'spec/acceptance'
 end
 
+desc "Run integration tests"
+RSpec::Core::RakeTask.new(:integration) do |t|
```

Comment:
Some modules already have a gem group called "system_tests". We should either make this project match them, or the other modules match this one.